packages/opensrp-plans/package.json

```diff
@@ -36,12 +36,20 @@
   },
   "peerDependencies": {
     "antd": "^4.6.3",
-    "react": "^16.18.1"
+    "react": "^16.18.1",
+    "react-redux": "^7.2.2",
+    "react-router": "^5.2.0",
+    "react-router-dom": "^5.2.0",
+    "redux": "^4.0.1"
   },
   "author": "OpenSRP Engineering",
   "license": "Apache-2.0",
   "private": false,
   "dependencies": {
-    "@opensrp/plan-form-core": "^0.0.1"
+    "@opensrp/plan-form-core": "^0.0.1",
+    "@onaio/redux-reducer-registry": "^0.0.9",
+    "@opensrp/react-utils": "^0.0.1",
+    "@opensrp/server-service": "^0.0.12",
+    "@opensrp/store": "^0.0.7"
```

Comment:
we've noticed adding opensrp-store as a dependencies results in duplicate stores which inturn breaks the page, I would suggest adding it to either peer or devDependencies depending on where its being used i.e. devDependencies if used only in tests.